iso/step_create.go

```diff
@@ -67,6 +73,7 @@ func (s *StepCreateVM) Run(_ context.Context, state multistep.StateBag) multiste
 		USBController:       s.Config.USBController,
 		Version:             s.Config.Version,
 		Firmware:            s.Config.Firmware,
+		Annotation:          s.Config.Annotation,
```

Comment:
interesting, we already have it in the driver.

Reply:
yeah i had the same reaction
https://github.com/jetbrains-infra/packer-builder-vsphere/commit/ba06b00e70d1c705835ea4894ec101e19f18b11b
I think you just forgot to copy/paste the line for annotation in this commit